.github/workflows/update_tutorials.yml

```diff
@@ -1,63 +1,75 @@
-name: update_tutorials
+name: Update tutorial workflows
 
 on:
   workflow_dispatch:
     inputs:
-      branch:
-        description: The branch to open the PR against
+      rose_ref:
+        description: The rose branch to update the tutorials on
+        required: false
+      cylc_ref:
+        description: The cylc-flow branch to update the tutorials from
         required: false
-        default: '2.0.x'
   schedule:
-  - cron: '37 04 * * 6' # 03:37, Saturday
-
-defaults:
-  run:
-    shell: bash
+    - cron: '37 04 6,21 * *' # 04:37, on the 6th & 21st of each month
 
 jobs:
-  create-tutorials-pr:
+  update_tutorials:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 5
+    env:
+      ROSE_REF: ${{ inputs.rose_ref || 'master' }}
+      CYLC_REF: ${{ inputs.cylc_ref || 'master' }}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - name: Checkout rose
+        uses: actions/checkout@v3
         with:
-          ref: ${{ github.event.inputs.rose_ref || github.sha }}
+          ref: ${{ env.ROSE_REF }}
           path: rose
 
-      - name: Update tutorials
-        run: |
-          git clone https://github.com/cylc/cylc-flow.git
-          git clone https://github.com/cylc/release-actions.git
-
-          # Two step process because rsync doesn't allow us
-          # to rename the folder.
-          cp -r cylc-flow/cylc/flow/etc/tutorial/cylc-forecasting-workflow \
-              cylc-forecasting-suite
+      - name: Checkout cylc-flow
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.CYLC_REF }}
+          path: cylc-flow
 
-          rsync -va cylc-forecasting-suite \
-              rose/metomi/rose/etc/tutorial/
       - name: Configure Git
         uses: cylc/release-actions/configure-git@v1
 
-      - name: Create PR
-        working-directory: rose
+      - name: Update tutorials
+        run: |
+          rsync -va cylc-flow/cylc/flow/etc/tutorial rose/metomi/rose/etc/tutorial \
+            --exclude=consolidation-tutorial \
+            --exclude=inheritance-tutorial \
+            --exclude=message-trigger-tutorial \
+            --exclude=retries-tutorial \
+            --exclude=runtime-introduction \
+            --exclude=runtime-tutorial
```

Comment:
I have included the whole tutorials dir, and blacklisted the subdirs not in rose.